src/Foundation/Base/code/Mappers/Mapper.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Wooli.Foundation.Base.Mappers
+{
+    using Sitecore.Diagnostics;
+
+    public abstract class Mapper : IMapper
+    {
+        protected AutoMapper.IMapper InnerMapper { get; set; }
```

Comment:
innerMapper

Reply:
We have Upper case naming of protected fields across the solution.